### DISCUSSION TITLE
ros1 lidar/enable_pointcloud service topic type deprecated

### DISCUSSION THREAD

**Discussion by Tifus15** [cyberbotics/webots#3928] [Ideas]
I have only a suggestion!
In ros1 with service  lidar/enable_pointcloud we get sensor_msgs::PointCloud topic. It's fine but it is already deprecated.
Could you convert this topic to PointCloud2 ?

Best Regards

Denis Andrić

**Comment by omichel** [comment]
We have PointCloud2 in the ROS 2 interface, see #3232.

**Reply by Tifus15** [reply]
I work explicitly in ROS1 for the Project. That was only a suggestion because I found solution with the converter. 
But thanks
